docs/generated/galasactl_auth_tokens_delete.md

```diff
@@ -0,0 +1,31 @@
+## galasactl auth tokens delete
+
+Deletes a personal access token
+
+### Synopsis
+
+Deletes a token used for authentication with the Galasa API server through the provided token id
+
+```
+galasactl auth tokens delete [flags]
+```
+
+### Options
+
+```
+  -h, --help             Displays the options for the 'auth tokens delete' command.
+      --tokenId string   The ID of the token to be deleted. This is an alphanumeric string
```

Comment:
Might not always be an alphanumeric string! (e.g. if an ID is `abcd-1234-xyz`, we should accept it)